app/controllers/application_controller.rb

```diff
@@ -2,6 +2,7 @@ class ApplicationController < ActionController::Base
   include Pundit
   before_action :authenticate_user!
   before_action :configure_permitted_parameters, if: :devise_controller?
+  # around_action :set_time_zone
```

Comment:
Does this need to be uncommented?

Reply:
Looks beautiful!  Just a couple thoughts as I poked around in there... 
1) We only operate during office hours (8am - 5pm), so there's really no need to list all hours of the day.  I'm afraid our less intuitive folks might feel like they need to set a Do Not Disturb time frame for each day of the year.  
2) There are two boxes for an "End Date".  I wasn't sure which one to use, as they both seem to do the same thing.

Reply:
@kyleboe yes, thank you! I'll do that.

@SarahOMD thanks for trying it out and providing your feedback!

1. I'll change the time dropdowns to 8am - 5pm as you suggested!
2. Can you clarify what you mean about the second 'End Date' box? There should be a 'Start Date' & 'End Date'. Additionally, if you configure a recurring blockout and change 'forever' to 'until' another date selection box appears where you can configure a date in the future where the recurrence ends. So for example, I could setup a blockout for where the start date is Wednesday, the end date is Friday, and it recurs every week until the end of next month.

Reply:
@kyleboe I've enable the `set_time_zone` around_action.

@SarahOMD I've updated the time dropdowns to 8am - 5pm and mentioned above :point_up: 

It's on staging now: http://omd-scheduler.herokuapp.com

Reply:
It looks great!!  Not confusing anymore :-) . Thanks!